backend/src/main/resources/application-local.properties

```diff
@@ -25,4 +25,5 @@ IEXCloud.secret.token=${IEXCLOUD_SECRET_TOKEN_SANDBOX}
 # Amazon S3 related properties
 aws.access.key.id=${AWS_ACCESS_KEY}
 aws.secret.access.key=${AWS_SECRET_ACCESS_KEY}
-aws.profile.pictures.bucket=${AWS_PROFILE_PICTURES_BUCKET}
\ No newline at end of file
+aws.profile.pictures.bucket=${AWS_PROFILE_PICTURES_BUCKET}
+aws.default.profile.picture=${DEFAULT_PROFILE_PICTURE}
```

Comment:
You do not need to store it as env variable, way too much work. The url does not need to be protected.
having `aws.default.profile.picture=https://my.default.url` is good enough

Reply:
sounds good